lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -42,8 +42,9 @@ __PACKAGE__->add_columns(
         data_type      => 'integer',
         is_foreign_key => 1,
     },
-    hidden => {
-        data_type     => 'boolean',
+    flags => {
```

Comment:
It's what we discussed in #837

Thing to consider though is if it is better to just add pinned column and have separate column for each feature or do it old school with multiple flags in one column.